Source/Diagnostics/ReducedDiags/FieldReduction.H

```diff
@@ -127,7 +127,7 @@ public:
         }
 
         // get parser
-        HostDeviceParser<m_nvars> reduction_function_parser = getParser(m_parser);
+        auto reduction_function_parser = m_parser->compile<m_nvars>();
```

Comment:
I am wondering if the many specializations in various TUs impact compile-time.
Will take a look when CI ran through just to get an idea.

Reply:
Haha, the error bar in CI is +/-5 min, so hard to say. Looks still the same :D